applications/DEMApplication/custom_constitutive/DEM_rolling_friction_model.cpp

```diff
@@ -1,8 +1,10 @@
-/////////////////////////////////////////////////
-// Main author: Chengshun Shang (CIMNE)
-// Email: chengshun.shang1996@gmail.com
-// Date: Aug 2022
-/////////////////////////////////////////////////
+//  Kratos Multi-Physics - ThermalDEM Application
+//
+//  License:       BSD License
+//                 Kratos default license: kratos/license.txt
+//
+//  Main authors:  Chengshun Shang (cshang@cimne.upc.edu)
```

Comment:
@ChengshunShang1996 is this the address you want to publish? You can also put two addresses. No problem from my side.

Reply:
please check other files where your address has been changed.

Reply:
Thanks! @maceligueta  When this address was modified, I was working together with @rlrangel. There is no problem with the address.